yt_dlp/options.py

```diff
@@ -408,6 +408,10 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help=(
             'Location of the main configuration file; either the path to the config or its containing directory '
             '("-" for stdin). Can be used multiple times and inside other configuration files'))
+    general.add_option(
+        '--plugin-locations',
+        dest='plugin_locations', metavar='PATH', action='append',
```

Comment:
I understand that the naming of this option was following the convention of `--ffmpeg-location` and `--config-location`, but I think following that convention is wrong to do here. Those two options allow the user to also specify the path to the **file**, not only the directory. Using the `-locations` name would imply that the user can specify the path to a wheel or zip, not just a directory. I think `--plugin-dirs` is better. It's shorter, it avoids confusion, it's more self-explanatory.

```suggestion
        '--plugin-dirs',
        dest='plugin_dirs', metavar='PATH', action='append',
```